Bulid errors on Debian -  The reference assemblies for framework ".NETFramework,Version=v4.7.2 were not found

I try to compile on Debian Testing and got a lot of errors like this:

The reference assemblies for framework ".NETFramework,Version=v4.7.2 not found. To resolve this, install the SDK or Targeting Pack for this framework version or retarget your application to a version of the framework for which you have the SDK or Targeting Pack installed.

There are two messages on Portuguese: one about v4.7.2 and another about v4.6.1:
```
/home/marcelo/.dotnet/sdk/3.1.408/Microsoft.Common.CurrentVersion.targets(1177,5): error MSB3644: os assemblies de referência para .NETFramework,Version=v4.7.2 não foram encontrados. Para resolver isso, instale o Pacote do Desenvolvedor (SDK/Pacote de Direcionamento) dessa versão do Framework ou redirecione seu aplicativo. É possível baixar Pacotes do Desenvolvedor do .NET Framework em https://aka.ms/msbuild/developerpacks [/home/marcelo/Downloads/ApsimX/APSIM.Shared/APSIM.Shared.csproj]
/home/marcelo/.dotnet/sdk/3.1.408/Microsoft.Common.CurrentVersion.targets(1177,5): error MSB3644: os assemblies de referência para .NETFramework,Version=v4.7.2 não foram encontrados. Para resolver isso, instale o Pacote do Desenvolvedor (SDK/Pacote de Direcionamento) dessa versão do Framework ou redirecione seu aplicativo. É possível baixar Pacotes do Desenvolvedor do .NET Framework em https://aka.ms/msbuild/developerpacks [/home/marcelo/Downloads/ApsimX/ApsimNG/ApsimNG.csproj]
/home/marcelo/.dotnet/sdk/3.1.408/Microsoft.Common.CurrentVersion.targets(1177,5): error MSB3644: os assemblies de referência para .NETFramework,Version=v4.7.2 não foram encontrados. Para resolver isso, instale o Pacote do Desenvolvedor (SDK/Pacote de Direcionamento) dessa versão do Framework ou redirecione seu aplicativo. É possível baixar Pacotes do Desenvolvedor do .NET Framework em https://aka.ms/msbuild/developerpacks [/home/marcelo/Downloads/ApsimX/Tests/UnitTests/UnitTests.csproj]
/home/marcelo/.dotnet/sdk/3.1.408/Microsoft.Common.CurrentVersion.targets(1177,5): error MSB3644: os assemblies de referência para .NETFramework,Version=v4.7.2 não foram encontrados. Para resolver isso, instale o Pacote do Desenvolvedor (SDK/Pacote de Direcionamento) dessa versão do Framework ou redirecione seu aplicativo. É possível baixar Pacotes do Desenvolvedor do .NET Framework em https://aka.ms/msbuild/developerpacks [/home/marcelo/Downloads/ApsimX/Models/Models.csproj]
/home/marcelo/.dotnet/sdk/3.1.408/Microsoft.Common.CurrentVersion.targets(1177,5): error MSB3644: os assemblies de referência para .NETFramework,Version=v4.6.1 não foram encontrados. Para resolver isso, instale o Pacote do Desenvolvedor (SDK/Pacote de Direcionamento) dessa versão do Framework ou redirecione seu aplicativo. É possível baixar Pacotes do Desenvolvedor do .NET Framework em https://aka.ms/msbuild/developerpacks [/home/marcelo/Downloads/ApsimX/Updater/Updater.csproj]
/home/marcelo/.dotnet/sdk/3.1.408/Microsoft.Common.CurrentVersion.targets(1177,5): error MSB3644: os assemblies de referência para .NETFramework,Version=v4.7.2 não foram encontrados. Para resolver isso, instale o Pacote do Desenvolvedor (SDK/Pacote de Direcionamento) dessa versão do Framework ou redirecione seu aplicativo. É possível baixar Pacotes do Desenvolvedor do .NET Framework em https://aka.ms/msbuild/developerpacks [/home/marcelo/Downloads/ApsimX/Docs/CreateDocumentation/CreateDocumentation.csproj]
```
I try to run  `./dotnet-install.sh --version 4.7.2` and got this messages:
```
dotnet-install: Note that the intended use of this script is for Continuous Integration (CI) scenarios, where:
dotnet-install: - The SDK needs to be installed without user interaction and without admin rights.
dotnet-install: - The SDK installation doesn't need to persist across multiple CI runs.
dotnet-install: To set up a development environment or to run apps, use installers rather than this script. Visit https://dotnet.microsoft.com/download to get the installer.

dotnet-install: Downloading primary link https://dotnetcli.azureedge.net/dotnet/Sdk/4.7.2/dotnet-sdk-4.7.2-linux-x64.tar.gz
curl: (22) The requested URL returned error: 404 
dotnet-install: The resource at https://dotnetcli.azureedge.net/dotnet/Sdk/4.7.2/dotnet-sdk-4.7.2-linux-x64.tar.gz is not available.
dotnet_install: Error: Could not find `.NET Core SDK` with version = 4.7.2
dotnet_install: Error: Refer to: https://aka.ms/dotnet-os-lifecycle for information on .NET Core support
```
Any clue for install?

Thank you so much!

--- a/Docs/content/Contribute/Compile/linux.md
+++ b/Docs/content/Contribute/Compile/linux.md
@@ -9,7 +9,7 @@ APSIM can be built against two target frameworks - .NET Framework 4.7.2 (which r
 
 2. Install the .NET Core 3.0 SDK. The [dotnet-install script](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script) is a simple way to do this
 
-2. Install required packages. (These commands and package names are for Ubuntu/Debian.)
+3. Install required packages. (These commands and package names are for Ubuntu/Debian.)
 
 	```sudo apt-get install <PackageName>```
 
@@ -24,7 +24,6 @@ APSIM can be built against two target frameworks - .NET Framework 4.7.2 (which r
 	- libwebkit1.1-cil
 	    
 		Note: this package is no longer available in Ubuntu's official repositories. It can be built from source, but can also be considered an optional dependency. The user interface will run without it, but some parts will be missing.
-
 	- webkit-sharp
 	
 		Note: this package is no longer available in Ubuntu's official repositories. If unavailable, run this command from the ApsimX directory:
@@ -38,13 +37,13 @@ APSIM can be built against two target frameworks - .NET Framework 4.7.2 (which r
 
 	To check if a package is already installed, use ```dpkg -s <PackageName>```
 
-3. Obtain source code
+4. Obtain source code
 
 	```git clone https://github.com/APSIMInitiative/ApsimX```
 
-4. Copy ApsimX/ApsimNG/Assemblies/Mono.TextEditor.dll.config to ApsimX/Bin/
+5. Copy ApsimX/ApsimNG/Assemblies/Mono.TextEditor.dll.config to ApsimX/Bin/
 
-5. Build the solution
+6. Build the solution
 
 	```dotnet build ApsimX.sln```
 
@@ -56,4 +55,14 @@ APSIM can be built against two target frameworks - .NET Framework 4.7.2 (which r
 
 	```dotnet build -f net472 ApsimX.sln```
 
+## Common Problems
 
+```
+/usr/share/dotnet/sdk/5.0.202/Microsoft.Common.CurrentVersion.targets(1216,5): error MSB3644: The reference assemblies for .NETFramework,Version=v4.6.1 were not found. To resolve this, install the Developer Pack (SDK/Targeting Pack) for this framework version or retarget your application. You can download .NET Framework Developer Packs at https://aka.ms/msbuild/developerpacks
+```
+
+This can be fixed by setting the FrameworkPathOverride environment variable. E.g.
+
+```bash
+FrameworkPathOverride=/usr/lib/mono/4.7.2-api/ dotnet build ApsimX.sln
+```

